README.rst

```diff
@@ -107,11 +256,6 @@ Configuration
 Operations
 ==========
 
-Bootstrapping Consumers
------------------------
-
-.. todo:: Rewrite this as part of the tutorial using pgbench after the replication worker is done.
-
```

Comment:
Oh yeah I knew that, I just wanted to actually move all the commands into a "demo" image or something like that.  I hadn't worked through it 100% but was aware of the example readme - I've referenced it a few times in the past few days so far.